lib/matplotlib/axes/_axes.py

```diff
@@ -3711,11 +3755,16 @@ def get_next_color():
 
         pc = PieContainer(slices, x, normalize)
 
-        if labeldistance is None:
+        if wedge_labels is not None:
+            self.pie_label(pc, wedge_labels, distance=wedge_label_distance,
+                           textprops=textprops)
+
+        elif labeldistance is None:
             # Insert an empty list of texts for backwards compatibility of the
             # return value.
             pc.add_texts([])
-        else:
+
+        if labeldistance is not None:
```

Comment:
Do we want to prohibit `wedge_labels is not None and labels is not None and labeldistance is not None`. This would add two sets of labels. Currently, and after deprecation, we'll only add one set of labels.

Since `wedge_labels` is new, anybody who sets that could also adapt to `labeldistance=None`; and if needed use `pie_label` to add the second set of labels.

Reply:
Done.